scripts/raw_to_easymocap.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, shutil
```

Comment:
Use `python-black` to format the source code

Reply:
Fixed with `python-black`